spirit/category/admin/templates/spirit/category/admin/index.html

```diff
@@ -15,8 +15,8 @@ <h1 class="headline">{% trans "Category list" %}</h1>
     <div class="rows">
 
         {% for c in categories %}
-            <div class="row">
-                <a href="{{ c.get_absolute_url }}">{{ c.title }}</a>
+            <div class="row" style="background-colour: {{ c.colour }}">
```

Comment:
uhm, `...-colour` is not a valid property (`...-color` is), but there should be no background...